gnosis-safe-deploy/scripts/proposeFortaAgent.js

```diff
@@ -0,0 +1,163 @@
+const EthersAdapter = require('@gnosis.pm/safe-ethers-lib').default;
+const Safe = require('@gnosis.pm/safe-core-sdk').default;
+const SafeServiceClient = require('@gnosis.pm/safe-service-client').default;
+const fortaAgent = require('forta-agent');
+
+const { ethers } = fortaAgent;
+
+// get the Awilix container with its registered modules
+const container = fortaAgent.configureContainer();
+
+// get the module corresponding to the agent registry Object
+// and the uploadImage function
+const {
+  agentRegistry,
+  uploadImage,
+  uploadManifest,
+  getCredentials,
+  agentId,
+  chainIds,
+} = container.cradle;
+
+// load values from the .env file
+require('dotenv').config();
+
+// load the address of our Gnosis Safe proxy smart contract
+const polygonSafeAddress = process.env.POLYGON_SAFE_ADDRESS;
+// load our JSON-RPC endpoint URL
+const polygonEndpoint = process.env.POLYGON_ENDPOINT;
+// load the private key for one of the owners of the Gnosis Safe
+const safeOwnerOnePrivateKey = process.env.OWNER_ONE_PRIVATE_KEY;
+
+async function main() {
+  // set up an ethers.js provider
+  const provider = new ethers.providers.JsonRpcProvider(polygonEndpoint);
+
+  // create an ethers.js Wallet (signer) and connect it to the provider
+  const safeWallet = new ethers.Wallet(safeOwnerOnePrivateKey);
```

Comment:
```suggestion
  const safeWallet = new ethers.Wallet(ownerPrivateKey);
```
Goes along with the above change.

Reply:
Updated.